plugins/catalog-backend/migrations/20201006130744_entity_data_column.js

```diff
@@ -46,20 +46,10 @@ exports.up = async function up(knex) {
       table.text('data').notNullable().alter();
     });
   }
-
-  // NOTE(freben): For some reason, specifically sqlite3 in-mem just drops some
-  // subset of constraints sometimes, when a table column is dropped - even if
-  // the column had no relation at all to the constraint. We therefore recreate
-  // the constraint here as a stupid fix.
-  if (knex.client.config.client === 'sqlite3') {
-    await knex.schema.alterTable('entities', table => {
-      table.unique(['full_name'], 'entities_unique_full_name');
-    });
-  }
```

Comment:
This is no longer necessary. In fact, if it is left in place, the migrations break. So this is the main risky bit with this update.